demos/can-validate/credit-card.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>can-validate Credit Card Demo</title>
+	<style>
+		.form-control {
+			display: block;
+		}
+
+		.form-control.has-errors {
+			border: 1px tomato solid;
+		}
+
+		form div {
+			margin: 1rem;
+		}
+		label small.has-errors {
+			color: tomato;
+		}
+	</style>
+</head>
+<body>
+	<script type="text/stache" id="demo-html">
+		<form>
+			<div>
+				<label>Credit Card Number <small class="has-errors">{{formatErrors(testSet('card', card), 'object').card.join(', ')}}</small></label>
```

Comment:
@Macrofig any chance you can update this example? Or at least confirm that it could be done the way @justinbmeyer is proposing?